containers/iam-reana-sync/README.md

```diff
@@ -0,0 +1,8 @@
+## Adding IAM users to Reana DB 
+
+This container inherits from the `reana-server` container to run DB operations. 
+At the moment, the users subscribe to IAM and get automatic access to Rucio. They can also get access to the [reana-vre.cern.ch](reana-vre.cern.ch) instance, but once inside they need to request a token, and their request needs to be approved by us after we receive an email notification. This is annoying, as with the IAM account they should automatically be able to use the VRE REANA cluster. This script therefore updates the email IAM list every day, to see if there are new users, and grants them a Reana token already, so they do not have to wait for us to grant them access to the instance. 
+
+The python scripts:
+-  `generate_email_list.py` fetches the email list of all IAM subscribed users and gets a .json list of all of them. In order to poll the IAM service to get all results, a workaround is needed as usually the basic script would only return the first 100 results. 
```

Comment:
Thanks for the detailed description. 
However, imo, 'small vulnerabilities' should not be described on the readme of a script (for us and fore external users). I suggest opening a(various) issue(s) to keep track of this.

Reply:
Also, from the text it is not clear if the workaround needs to be implemented, or it's already done.